src/store/selectors/stories/index.ts

```diff
@@ -690,19 +695,31 @@ function createVerifyMemberStory(
 
 function createTipCallToAction(
   member: Member,
+  targetOperationId: OperationId,
   tips?: TipGiveOperation[]
 ): CallToAction | undefined {
-  if (!tips) {
-    return undefined;
+  if (!tips || tips.length === 0) {
+    return [
+      {
+        type: CallToActionDataType.TIP,
+        data: {
+          tipTotal: new Big(0),
+          targetOperationId,
+          fromMemberIds: new Set<MemberId>(),
+          toMemberId: member.get("memberId")
+        }
+      }
+    ];
   }
 
+  const toMemberId = member.get("memberId");
   const tipData: TipData = tips.reduce(
     (data, tip) => {
-      if (tip.data.to_uid === member.get("memberId")) {
+      if (tip.data.to_uid === toMemberId) {
         return {
-          tipTotal: data.tipTotal.plus(tip.data.donation_amount),
-          fromMemberIds: data.fromMemberIds.concat(tip.creator_uid),
-          toMemberId: data.toMemberId
+          ...data,
+          tipTotal: data.tipTotal.plus(tip.data.amount),
```

Comment:
Could we make this the total tip amount including donated amount? I think similar to give notifications, it makes sense for the displayed amount to be total tipped.

Reply:
Do you think I need to display donated amount separately somewhere then?